python/py-ukpostcodeparser/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-ukpostcodeparser
+set real_name       UkPostcodeParser
```

Comment:
please don't use your own variable here; do `pythont.rootname UkPostcodeParser` instead so that the python PG will do the rest of the magic (see below)

Reply:
Thanks, done!